src/internal/timer.rs

```diff
@@ -1,6 +1,7 @@
 use chrono::{DateTime, Duration, Utc};
 use std::time::Duration as StdDuration;
 use std::thread;
+use std::{i64};
```

Comment:
Unnecessary import.

Reply:
Idk why but my ide was complaining when it wasn't imported, but I can remove it again